mesonbuild/mesonlib.py

```diff
@@ -21,6 +21,21 @@
 import collections
 from mesonbuild import mlog
 
+have_fcntl = False
+have_msvcrt = False
+
+try:
+    import fcntl
+    have_fcntl = True
+except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [SideCI](https://sideci.com) -->